### DISCUSSION TITLE
[14일차-윤경]flex와 grid의 차이

### DISCUSSION THREAD

**Discussion by skyhanull** [zzangCS/zzangInterview#46] [프론트 면접 질문]
flex와 grid의 차이에 대해 말씀해주세요

**Comment by skyhanull** [comment]
flex는 한 줄로 된 행 혹은 열 방향의 레이아웃과 같이 일차원의 레이아웃을 염두에 두고 설계되었습니다 
grid는 행과 열 모두를 염두에 둔 2차원적 레이아웃을 고려해서 설계되었습니다.

**Comment by ez1n** [comment]
grid : 컨테이너
⇒ 항목을 두 방향으로 레이아웃 할 수 있다.
⇒ 레이아웃 위주의 정렬

flex : 컨텐츠
⇒ 열 또는 행을 한 방향으로만 레이아웃(정렬) 할 수 있다.
⇒ 컨텐츠 위주의 정렬

복잡한 레이아웃 설계가 필요한 경우 grid가 효과적
